### PR TITLE
Central Server deprecation

### LINKED ISSUE #327: Disable new device registrations (except One-Click)

### PR DESCRIPTION
**Pull request by benjaoming**
Fixes #327 #328 #329 

1. [x] Omnipresent notification text, please add suggestions in this PR
1. [x] Removal of new signups (only One-Click Registration allowed)

CC: @carinediaz @jamalex 

image

### CODE REVIEW

**Comment by benjaoming** [comment]
Here is the flow of the registrations being disabled. Note that step 1 cannot be changed and step 2 is unchanged from previously - I thought it's better to dedicate a separate page for the "Registration closed" message.

1) In KA Lite, people click this button after a fresh installation...

image

2) Now, they end up at this page and should click "sign up here":

image

3) After which they are told that registration is closed.

image

**Comment by benjaoming** [comment]
I am fully aware that all the lines are to long to be reader-friendly, but I'm not going to fix the Stylesheets on the central server. All the layouts are somewhat doubtful in hindsight.

**Comment by benjaoming** [comment]
Main landing page on kalite.learningequality.org

image

File: centralserver/central/templates/central/base.html [hunk 1]
```diff
@@ -158,6 +158,24 @@
...
+                        <h3>This service is being replaced by <a href="https://learningequality.org/kolibri/" target="_blank">Kolibri</a>.</h3>
+                        <p>
+                            You can export your data until <strong>March 1st 2020</strong>
+                            after which, this service will be taken down.
+                            KA Lite is fully replaced by a more modern, mature, stable and featureful
+                            platform called <em>Kolibri</em>. It is free and Open Source.
+                            By using Kolibri, you will have access to more content, not least newer and more plentyful Khan Academy content.
+                            You will also have online synchronization and data export with
+                            Kolibri Data Hub, to be launched in early 2020.
+                        </p>
```
Comment (carinediaz) [inline centralserver/central/templates/central/base.html]: **Important announcement: KA Lite Central Server deprecation** 

The KA Lite Central Server is in the process of being deprecated.
**You have until **March 1st, 2020** to export your data**. After that date, the service will be completely taken down. 
KA Lite is no longer being actively developed, and it's being replaced by [Kolibri](https://learningequality.org/kolibri/), a free and open-source, second-generation platform that contains all of KA Lite videos and exercises, and many, many more content sources and pedagogical tools. You can [test it here](https://kolibridemo.learningequality.org/en/user/#/signin), and [download it here](https://learningequality.org/download/). If you do require a similar central data syncing service, please contact us via info@learningequality.org, so we can best assess your needs and how we can support.

Comment (carinediaz) [reply centralserver/central/templates/central/base.html]: @benjaoming Please let me know if you have any comments. Jamie also already reviewed it and okayed it. Thank you!

Comment (benjaoming) [reply centralserver/central/templates/central/base.html]: Thanks so much @carinediaz ! I have implemented these changes.

Comment (benjaoming) [reply centralserver/central/templates/central/base.html]: Thanks so much @carinediaz ! I have implemented these changes.

**Pull request merged**